app/uploaders/hero_image_uploader.rb

```diff
@@ -2,14 +2,18 @@
 
 # app/uploaders/hero_image_uploader.rb
 class HeroImageUploader < DefaultUploader
-  process resize_to_fit: [2260, 700]
+  process resize_to_fill: [2260, 700]
```

Comment:
I'm assuming this is in response to images not being full width, but is it safe to crop here if we don't supply the images?

Reply:
I mean we have to as this is the size of the area it takes on the page anyway - so the crop is being done in browser either way